inc/colours.php

```diff
@@ -174,4 +174,42 @@ function hale_get_colour_hint($colour_array_item) {
 	function hale_get_colour_options($colour_array_item) {
 		return $colour_array_item[4];
 	}
-?>
+
+	function hale_new_colour_check() {
+		// This function checks for any newly created colours that haven't been set in the customizer and applies the default.
+		// The amended CSS file will be overwritten when the colours are next amended.
+		// This does not protect against new colours being added to the SASS and not added to colours.php.
+		// This is for non-IE only as this particular bug doesn't occur in IE.
+
+		$CSSfileURL = wp_get_upload_dir()["basedir"]."/custom-colours.css"; //non-IE CSS file (variable values)
+		$ColourFileURL = get_template_directory()."/inc/colours.php"; //colours CSS file
```

Comment:
Should this be a css file not the php file?

Reply:
The PHP file contains the latest colours, so we need to check the CSS file (184) with the PHP file (185).  If the CSS file is newer, then the code that created it ran with the latest colours so we can be confident that it is up-to-date.